### ISSUE TITLE
Quais será as jornadas de usuario? Locatario?Locador?Como elas interagem?

### ISSUE THREAD

**Issue by gilberto-009199** [EdsonMostaco/ParkingSpot#2]
Olá, @EdsonMostaco 
Como vc imagina o Locatario e o Locador interagindo? Vai ter um chat entre eles? Vai ter avaliação?

**Comment by EdsonMostaco** [comment]
O locatário poderá cadastrar suas vagas na aplicação e disponibilizar para que outros usuário vejam o local com seus detalhes e o preço pedido.
O locador acessa a ferramenta para buscar uma garagem e inicia o processo de contratação on-line. O pagamento da transação será com cartão de crédito. 
Podemos implementar um chat entre o locador e o locatário para que possam tirar maiores dúvidas.
Sim, é importante ter um sistema de avaliação para ambos os lados.

**Comment by gilberto-009199** [comment]
Quais são os dados de uma locação? e quais serão os dados de um usuario e um carro?

**Comment by gilberto-009199** [comment]
quais dados precisamos guardar de cada um

**Comment by gilberto-009199** [comment]
Precisamos validar documentos?Digamos CNH, RG e etc?

**Comment by gilberto-009199** [comment]
Quais as validações que um  Locatário precisa?
+ CPF(Precisamos validar na receita federal?)
+ Precisamos de um mecanismo de validar documentos? A pessoa vai precisar tirar uma foto segurando o documento?
+ Precisamos que o locatario tire fotos junto com o carro?

**Comment by EdsonMostaco** [comment]
> quais dados precisamos guardar de cada um

usuário simples:
- email
- senha

locatários:
-dados pessoais
--nome;
--RG;
--CPF;
--Endereço;
--Dados da Garagem
---Endereço;
---Coberta -> sim ou não;
---Condomínio -> sim ou não;
---comprimento;
---largura;
---altura;
---preço desejado;

locador:
-dados pessoais
--Nome
--RG
--CPF
--Endereço
--Dados do veículo
---marca
---modelo
---ano
---cor
---nome do dono no documento

A principio estes são os dados que pensei, mas acredito que talvez precise de mais informações para intermediar uma locação.

**Comment by EdsonMostaco** [comment]
> Quais as validações que um Locatário precisa?
> 
> * CPF(Precisamos validar na receita federal?)
> * Precisamos de um mecanismo de validar documentos? A pessoa vai precisar tirar uma foto segurando o documento?
> * Precisamos que o locatario tire fotos junto com o carro?

Sim, será necessário um compliance forte e rigido com consultas de cpf e até mesmo consulta de mandados de busca e apreensão veicular, mandados de prisão contra o locador, validação antifraude nas documentações enviadas. 
Fotos do veiculo é uma ideia, pois o veiculo ficará sob a guarda de um terceiro. Além disso serve de confirmação das informações.

**Comment by gilberto-009199** [comment]
Será que um locador comum que deseja alugar uma garagem precisa de todos esses dados? Ou será que ele vai querer passar todos esses dados só apra alugar?
Não seria mais simples:
 + CPF(Validado na receita federal)
 + cartão de credito
 + Nome
 + Foto 
 + Celular(Confirmação SMS)
 + Foto do RG ou CNH
